src/main/java/org/broadinstitute/hellbender/engine/FeatureContext.java

```diff
@@ -242,11 +241,11 @@ private int windowStop(final SimpleInterval locus, final int windowTrailingBases
      *         this FeatureContext's query interval. Empty List if there is no backing data source and/or interval.
      */
     public <T extends Feature> List<T> getValues(final Collection<FeatureInput<T>> featureDescriptors) {
-        if (featureManager == null || interval == null) {
+        if (featureManager == null || interval == null || featureDescriptors.isEmpty()) {
```

Comment:
I think he rebased a couple of Adam's commits, actually -- recommend repairing the branch by removing these commits from its history during the next `rebase -i`